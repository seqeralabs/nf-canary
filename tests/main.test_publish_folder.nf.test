nextflow_process {

    name "Test Process TEST_PUBLISH_FOLDER"
    script "main.nf"
    process "TEST_PUBLISH_FOLDER"

    test("TEST_PUBLISH_FOLDER") {

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
        }

    }

}
