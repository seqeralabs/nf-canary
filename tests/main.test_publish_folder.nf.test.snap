{
    "TEST_PUBLISH_FOLDER": {
        "content": [
            {
                "0": [
                    [
                        "test1.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                        "test2.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-04-18T17:19:30.136869"
    }
}